doc/integrator/tilegeneration.rst

```diff
@@ -116,19 +112,16 @@ https://github.com/camptocamp/tilecloud-chain/blob/master/tilecloud_chain/scaffo
 The main thing to do is to:
 
 * Set the resolutions we want to generate in the ``grids``.
-  If we want to generate different resolution per layers we should create
-  deferent grid.
+  If we want to generate different resolution per layers we should create deferent grid.
```

Comment:
"If we want to generate different resolutions par layer, we should create different grids."